Tasks/PublishBuildArtifacts/publishbuildartifacts.ts

```diff
@@ -5,6 +5,14 @@ import tl = require('vsts-task-lib/task');
 
 tl.setResourcePath(path.join(__dirname, 'task.json'));
 
+// Getting host type, throwing error if it is invoked through Release
```

Comment:
This task has L0's that should be kept up to date. That could be a follow-up change so this doesn't miss sprint cut-off. But they should be updated.